src/stylesheets/_resource_empty_tile.scss

```diff
@@ -0,0 +1,15 @@
+.resource-empty-state {
```

Comment:
Shouldn't this file be named _resource_empty_state.scss ?

Reply:
updated